docs/changelog.md

```diff
@@ -10,6 +10,7 @@
     * Dusty now attempts to fix a known problem with boot2docker networking, resulting in up to a 10x improvement in network performance. See [this boot2docker issue](https://github.com/boot2docker/boot2docker/issues/1022) for more information.
 * **Misc**
     * Testing `once` commands that fail now cause the entire test run to fail immediately.
```

Comment:
Drive-by, can you rephrase this? It sounds like you are literally testing the once commands